tsconfig.json

```diff
@@ -3,11 +3,17 @@
       "target": "es6",
       "module": "commonjs",
       "outDir": "./dist",
+      "moduleResolution": "node",
       "strict": true,
+      "resolveJsonModule": true,
       "sourceMap": true,
       "esModuleInterop": true,
       "skipLibCheck": true,
-      "forceConsistentCasingInFileNames": true
+      "forceConsistentCasingInFileNames": true,
+      "baseUrl": "./src",
+      "paths": {
+        "@/*": ["*"]
+      }
     },
```

Comment:
` 
   "rootDir": "./src",
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"]
    },
`
ofcourse each one will be on a new line..not straight